tests/acceptance/features/apiSharingNg/shareInvitations.feature

```diff
@@ -96,3 +96,95 @@ Feature: Send a sharing invitations
       | Co Owner    | folder        | FolderToShare  |
       | Uploader    | folder        | FolderToShare  |
       | Manager     | folder        | FolderToShare  |
+
+
+  Scenario Outline: send sharing invitation to group with different roles via the Graph API
+    Given user "Carol" has been created with default attributes and without skeleton files
+    And group "grp1" has been created
+    And the following users have been added to the following groups
+      | username | groupname |
+      | Brian    | grp1      |
+      | Carol    | grp1      |
```

Comment:
It would be great also to check that users have share using `me/drive/sharedWithMe`

we can implement this steps in another PR after merging https://github.com/owncloud/ocis/pull/8010